plugins/woocommerce/client/blocks/packages/components/totals/item/index.tsx

```diff
@@ -14,7 +14,7 @@ import FormattedMonetaryAmount from '../../formatted-monetary-amount';
 
 export interface TotalsItemProps {
 	className?: string | undefined;
-	currency: Currency;
+	currency?: Currency | undefined;
```

Comment:
```suggestion
	currency?: Currency;
```
I think both additions (`?` and `| undefined`) are doing the same thing?

Reply:
having `currency?: Currency | undefined;` allows the currency prop to be explicitly set as undefined, e.g.

```jsx
<TotalsItemProps currency={ undefined } ... />
```

whereas just `currency?: Currency;` does not allow this, it either has to be missing, or `Currency`.

```jsx
<TotalsItemProps currency={ someCurrencyObject } ... />
```

or

```jsx
<TotalsItemProps ... />
```

Reply:
My understanding is that optional (`?`) still allows you to set it explicitly as `undefined`. See [this example](https://www.typescriptlang.org/play/?#code/C4TwDgpgBAKhDOwoF4oG8CwAoKUD2YwAlngHYCGANgPwBcUiATkaQOYDc2AvttgPR8oASSgBjcqQYQkRJBAAeYSkVGzKIKOXhQArqQAmEAGYsI+7KLKJN9ONdSYc+QiQqV6ewydJnuvLAL4jAwA1kRgULIWVkgARrYISA48WNigkFAAgqR4wAAWEIx2SejYuATEZFT0TCysUAA+ugbGpvqcWCn8giLikvDSkXKKyqrA6pranq0+5liWpNbkAEz02bkFRYkopU4VrtXNXm1+qQGCfQDkSPBhEfl4Oqx50Ytxq1k5+YXFO2hcQA).

Reply:
I get an error with this change..

```
Type 'Currency | undefined' is not assignable to type 'Currency'.
```